src/firefly/js/ui/SuggestBoxInputField.jsx

```diff
@@ -260,7 +260,7 @@ export class SuggestBoxInputFieldView extends PureComponent {
     render() {
 
         const {displayValue, valid, message, highlightedIdx, isOpen, inputWidth, suggestions, mouseTrigger } = this.state;
-        const {label, tooltip, inline, renderSuggestion, wrapperStyle, placeholder, slotProps={},
+        const {label, tooltip, inline, renderSuggestion, wrapperStyle, placeholder, slotProps={}, sx,
```

Comment:
at some point we should remove wrapperStyle. I have been taking it out of other components.